aws-lc-rs/src/digest/sha.rs

```diff
@@ -143,3 +193,17 @@ fn sha512_256_digest(msg: &[u8], output: &mut [u8]) {
         aws_lc::SHA512_256(msg.as_ptr(), msg.len(), output.as_mut_ptr());
     }
 }
+
+fn sha3_384_digest(msg: &[u8], output: &mut [u8]) {
+    let mut ctx = Context::new(&SHA3_384);
+    ctx.update(msg);
+    let digest = ctx.finish();
+    output[0..SHA3_384_OUTPUT_LEN].copy_from_slice(digest.as_ref());
+}
+
+fn sha3_512_digest(msg: &[u8], output: &mut [u8]) {
+    let mut ctx = Context::new(&SHA3_512);
+    ctx.update(msg);
+    let digest = ctx.finish();
+    output[0..SHA3_512_OUTPUT_LEN].copy_from_slice(digest.as_ref());
+}
```

Comment:
I couldn't find a "one-shot" digest function for SHA3 in AWS-LC.

Reply:
Yeah I looked around and there doesn't seem to be a "one-shot" SHA3 digest function in OpenSSL either.